addons/hr_material/data/hr_material_data.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <!-- Standard stages for Maintenance Request -->
+        <record id="stage_new" model="hr.material.stage">
+            <field name="name">New Request</field>
+            <field name="sequence" eval="1" />
+            <field name="fold" eval="False" />
+        </record>
+        <record id="stage_in_progress" model="hr.material.stage">
+            <field name="name">In Progress</field>
+            <field name="sequence" eval="2" />
+            <field name="fold" eval="False" />
+        </record>
+        <record id="stage_repaired" model="hr.material.stage">
+            <field name="name">Repaired</field>
+            <field name="sequence" eval="3" />
+            <field name="fold" eval="True" />
+        </record>
+        <record id="stage_scrap" model="hr.material.stage">
+            <field name="name">Scrap</field>
+            <field name="sequence" eval="4" />
+            <field name="fold" eval="True" />
+        </record>
+        <record id="stage_cancel" model="hr.material.stage">
+            <field name="name">Cancelled</field>
+            <field name="sequence" eval="10" />
+            <field name="fold" eval="True" />
+        </record>
+
+        <record id="mail_alias_material" model="mail.alias">
+            <field name="alias_name">internal.material</field>
```

Comment:
Defining a default alias is risky. If people do not pay attention they will have emails going to this address that is not necessarily configured in their email servers. Is it really necessary ? It should be blank by default IMHO.